torchaudio/backend/_soundfile_backend.py

```diff
@@ -103,11 +103,15 @@ def load(
         format (str, optional):
             Not used. PySoundFile does not accept format hint.
 
-    Returns:
-        torch.Tensor:
-            If the input file has integer wav format and normalization is off, then it has
-            integer type, else ``float32`` type. If ``channels_first=True``, it has
-            ``[channel, time]`` else ``[time, channel]``.
+    :rtype:
+            tuple(torch.Tensor, int)
+    :returns:
+            torch.Tensor:
+                If the input file has integer wav format and normalization is off, then it has
+                integer type, else ``float32`` type. If ``channels_first=True``, it has
+                ``[channel, time]`` else ``[time, channel]``.
+            int:
+                Sample rate of the audio.
```

Comment:
Could you keep the `Returns:` directive for the consistency with other docstrings?

Something like the following.

```rst
    Returns:
        Tuple[torch.Tensor, int]: Resulting Tensor and sample rate.
             If the input file has integer wav format and normalization is off, then it has
             integer type, else ``float32`` type. If ``channels_first=True``, it has
             ``[channel, time]`` else ``[time, channel]``.
```

Reply:
Done